frontend/website/docs/troubleshooting.md

```diff
@@ -15,20 +15,19 @@ Follow the steps below to use [MiniUPnP](https://github.com/miniupnp/miniupnp) t
         $ ifconfig
         ...
         en0: flags=8863<UP,BROADCAST,SMART,RUNNING,SIMPLEX,MULTICAST> mtu 1500
-                ether 8c:85:90:c9:a2:01 
```

Comment:
If we don't need the ip address, can you remove this section as well?

Reply:
done.